cmake/traccc-compiler-options-cpp.cmake

```diff
@@ -22,8 +22,10 @@ if( ( "${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU" ) OR
    traccc_add_flag( CMAKE_CXX_FLAGS "-Wextra" )
    traccc_add_flag( CMAKE_CXX_FLAGS "-Wshadow" )
    traccc_add_flag( CMAKE_CXX_FLAGS "-Wunused-local-typedefs" )
-   traccc_add_flag( CMAKE_CXX_FLAGS "-pedantic" )
-   traccc_add_flag( CMAKE_CXX_FLAGS "-Wfloat-conversion" )
+   traccc_add_flag( CMAKE_CXX_FLAGS "-Wpedantic" )
```

Comment:
:confused: No, it was not a typo. `-pedantic` is the correct flag for GCC and LLVM/Clang.

Reply:
I was under the impression `-pedantic` is a C thing, but ok. `-Wpedantic` is still much more common.

Reply:
I **think** `-pedantic` is "more pedantic" than `-Wpedantic`.

https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html

We definitely use `-pedantic` in all of the ATLAS builds since forever. :thinking:

Reply:
ok

Reply:
```suggestion
```